lib/unobtainium/support/runner.rb

```diff
@@ -156,10 +156,7 @@ def destroy
       private
 
       def close(channel)
-        if channel.nil?
-          return
-        end
-        channel.close
+        channel&.close
```

Comment:
This new Ruby feature (how is it called again) might not be available in older ruby versions.